src/utils/helpers.js

```diff
@@ -48,8 +48,12 @@ export const toEntity = (data, entityType, idKey = 'address') => {
  */
 export const weiToEth = (value) => {
   let ethValue = '0'
-  if (value && Decimal(value).gt(0)) {
-    ethValue = Decimal(value).div(1e18).toString()
+  if (value) {
+    if (value.gt && value.gt(0)) {
```

Comment:
Hmm, if the value has the `.gt`method but it is not greater than 0, it will try to parse it to a new decimal? What if it is a negative number?

To typecheck here for decimal I'd rather use `typeof value === 'string'` and then parse to decimal

Reply:
right